### PR TITLE
Sam Cogan case study

### PR DESCRIPTION
**Pull request by gthuang**
## Description
A case study about Sam Cogan and his testing practices. [Google Doc](https://docs.google.com/document/d/1yRdtL0z2peal7Mtu1c9afqTh0uvyXth4BeSKTABm0f4/edit).

## Checklist:

- [x] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [x] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [x] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

**Review by scottslowe — APPROVED** [review]
LGTM

**Comment by gthuang** [comment]
@interurban @scottslowe please don't merge until @aaronkao approves as this might get another revision pending additional feedback

**Pull request merged**
